[2025-03-21T13:31:23.979+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T13:31:24.013+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2025-03-21T13:31:08.403138+00:00 [queued]>
[2025-03-21T13:31:24.030+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2025-03-21T13:31:08.403138+00:00 [queued]>
[2025-03-21T13:31:24.030+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-21T13:31:24.050+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2025-03-21 13:31:08.403138+00:00
[2025-03-21T13:31:24.064+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=637) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-21T13:31:24.067+0000] {standard_task_runner.py:72} INFO - Started process 639 to run task
[2025-03-21T13:31:24.068+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2025-03-21T13:31:08.403138+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpgqqptlkj']
[2025-03-21T13:31:24.073+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask local_to_gcs_task
[2025-03-21T13:31:24.143+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2025-03-21T13:31:08.403138+00:00 [running]> on host 4791d3610749
[2025-03-21T13:31:24.279+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='local_to_gcs_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T13:31:08.403138+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T13:31:08.403138+00:00'
[2025-03-21T13:31:24.281+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T13:31:25.023+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T13:31:25.039+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T13:31:25.040+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, run_id=manual__2025-03-21T13:31:08.403138+00:00, execution_date=20250321T133108, start_date=20250321T133124, end_date=20250321T133125
[2025-03-21T13:31:25.127+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T13:31:25.162+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-21T13:31:25.164+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
